taichi/codegen/codegen_llvm.cpp

```diff
@@ -1074,28 +1074,35 @@ void CodeGenLLVM::visit(ReturnStmt *stmt) {
   if (stmt->ret_type.is_pointer()) {
     TI_NOT_IMPLEMENTED
   } else {
+    TI_ASSERT(
+        stmt->values.size() <= taichi_max_num_ret_value &&
+        "ERROR: Return list's size must less than or equal to 30 currently!");
```

Comment:
1. Why `&&`?
2. 30 should be replaced with `taichi_max_num_ret_value`.

Reply:
3b357e2726e185846d448ff4bd6a00b039c5c79b